CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 10.1.0
+*  Moved the CampaignReportToCsv example to the Migration/ directory.
```

Comment:
Just noticed that the indentation is not the exact same as others (missing spaces).

Reply:
Done.